internal/service/ipam/model_ipamsvc_ip_space.go

```diff
@@ -29,6 +29,7 @@ type IpamsvcIPSpaceModel struct {
 	AsmConfig                       types.Object      `tfsdk:"asm_config"`
 	AsmScopeFlag                    types.Int64       `tfsdk:"asm_scope_flag"`
 	Comment                         types.String      `tfsdk:"comment"`
+	CompartmentId                   types.String      `tfsdk:"compartment_id"`
```

Comment:
can you also add this new fields to the datasource test as well. Comment applies to the other models as well

https://github.com/infobloxopen/terraform-provider-bloxone/blob/ec7ab6247b2631b1df26bd78e24aa73b54f13150/internal/service/ipam/api_ip_space_data_source_test.go#L61

Reply:
Done